include/services/OswServiceTaskBLEServer.h

```diff
@@ -67,6 +67,75 @@ class OswServiceTaskBLEServer : public OswServiceTask {
 
         OswServiceTaskBLEServer* task;
     };
+    class StepsTodayCharacteristicCallbacks: public NimBLECharacteristicCallbacks {
+        void onRead(NimBLECharacteristic* pCharacteristic);
+        /*
+            uint32_t value;
+            this->bytes[0] = (uint8_t) value; 
+            this->bytes[1] = (uint8_t) (value >> 8);
+            this->bytes[2] = (uint8_t) (value >> 16);
+            this->bytes[3] = (uint8_t) (value >> 24);
+        */
```

Comment:
Please rework these comments to express what they mean: this is a 4-byte array of a `uint_32_t` number